src/components/islands/search.jsx

```diff
@@ -222,7 +222,7 @@ function SearchResult({ result = [], cursor, searchTerm }) {
   const router = useRouter();
 
   return (
-    <div className="block w-full text-sm md:text-base">
+    <div className="block w-full text-sm md:text-base overflow-y-scroll">
```

Comment:
```suggestion
    <div className="block w-full text-sm md:text-base overflow-y-scroll scrollbar">
```

We've got this utility class `scrollbar` that we use for our vertical scrollbar to make it some bit of uniform around the project. Let's use that here.

See it in action...

[screencast-bpconcjcammlapcogcnnelfmaeghhagj-2024.09.14-10_45_16.webm](https://github.com/user-attachments/assets/dfb650cc-7920-4a56-adb6-2c37b8a0da50)